linera-service/src/database_tool.rs

```diff
@@ -234,8 +263,11 @@ async fn main() {
 
     let options = DatabaseToolOptions::from_args();
     let error_code = match evaluate_options(options).await {
-        Ok(_) => 0,
-        Err(_) => 2,
+        Ok(code) => code,
+        Err(msg) => {
+            println!("Error is {:?}", msg);
+            2
```

Comment:
Can we return 1 here (as before) instead of 2?

Reply:
Well, the original proposal of Christos (that you seemed to agree with) was that:
* `0` means that the table exists.
* `1` means that the table is absent
* `_` means that another error happens.

The `_` I could only provide with a `2`. But if we put `1` then we merge both error messages and that is a loss of information which I find unfortunate.

Reply:
I'm just suggesting to swap 1 and 2

Reply:
Probably not super important

Reply:
Ah, ok. Well, I would follow Christos in that case.